homepage/homepage/content/docs/server-side/setup.mdx

```diff
@@ -117,3 +117,34 @@ Currently, the Jazz Loader is tested on the following edge environments:
 
 - Edge runtime environment that supports WebAssembly
 - `jazz-tools/load-edge-wasm` must be imported before any Jazz import
+
+
+## N-API
+
+Jazz supports a native Node.js implementation of its core crypto, but it is not enabled by default.
```

Comment:
```suggestion
Jazz uses a WASM-based crypto implementation for maximum compatibility and performance across a wide variety of environments. 

If you’re in full control of your runtime and know it will always run on Node.js with a supported hardware configuration, you can enable the faster native Node.js crypto implementation instead.
```